ch03.adoc

```diff
@@ -295,7 +295,7 @@ The attributes **`flag_values`**,
 
 The **`flag_values`** and **`flag_meanings`**
       attributes describe a status flag consisting of mutually exclusive coded values. The **`flag_values`** attribute is the same type as the variable to which it is attached, and contains a list of the possible flag values. The **`flag_meanings`** attribute is a string whose value is a blank separated list of descriptive words or phrases, one for each flag value. Each word or phrase should consist of characters from the alphanumeric
-      set and the following five: '_', '-', '.', '+', '@'. If multi-word phrases are used to describe the flag values, then the words within a phrase should be connected with underscores. The following example illustrates
+      set and the following five: "_","-", ".","+", "@". If multi-word phrases are used to describe the flag values, then the words within a phrase should be connected with underscores. The following example illustrates
```

Comment:
> single quotes in asciidoc give emphasised text with no quotes

As of asciidoctor 1.5 (as used by the automated build) this is [no longer true](https://github.com/asciidoctor/asciidoctor/issues/717). You can see this at http://cf-metadata.github.io/#flags. So the simplest thing to do here is nothing.

If you're stuck with an earlier version then perhaps the back-tick - forward-tick technique (as described for ["Single quoted text"](http://www.methods.co.nz/asciidoc/chunked/ch10.html)) would allow it to work in both versions.